src/utility/Adafruit_GFX.cpp

```diff
@@ -160,7 +160,7 @@ void Adafruit_GFX::fillCircleHelper(int16_t x0, int16_t y0, int16_t r,
   }
 }
 
-// Bresenham's algorithm - thx wikipedia
+// Bresenham's algorithm - thx wikpedia
```

Comment:
This is supposed to be "Wikipedia".
Instead make thx to Thanks or even better, try following official reference guide -
https://elearning.unito.it/mooc/pluginfile.php/221/mod_resource/content/1/Reference%20Guide%20S.L.%20McGaughey%20%282004%29.pdf

Reply:
requested changes have been done.